tools/localExec.py

```diff
@@ -31,10 +31,12 @@ def __init__(self,desc,options={}):
     ## Helpers Functions ##
     # The set of input parameters from the json descriptor
     self.inputs     = self.desc_dict['inputs'] # Structure: [ {id:...},...,{id:...} ]
+    # The set of output parameters from the json descriptor
+    self.outputs     = self.desc_dict['output-files'] # Structure: [ {id:...},...,{id:...} ]
     # The set of parameter groups, according to the json descriptor
     self.groups     = self.desc_dict['groups'] if 'groups' in self.desc_dict.keys() else []
     # Retrieves the parameter corresponding to the given id
-    self.byId       = lambda n: [ v for v in self.inputs if v['id']==n][0]
+    self.byId       = lambda n: [ v for v in self.inputs+self.outputs if v['id']==n][0]
```

Comment:
This one worries me a bit, since previously I assumed I could only get inputs from it. But, looking through the script, I think it is ok.